404.html

```diff
@@ -3,25 +3,97 @@
 permalink: /404.html
 ---
 <div class="container">
-    <!--Error Block-->
-    <div class="row">
-        <div class="col-md-8 col-md-offset-2">
-            <div class="error-v1">
-                <span class="error-v1-title">404</span>
-                <span>Page not found!</span>
-                <p>The requested URL was not found. </p>
-            </div>
-        </div>
+  <div class="row">
+    <div class="col-md-8 col-md-offset-2">
+      <h1>Page not found</h1>
+
+      <p>Sorry, we could not find the page you were looking for. You can <a href="/">go to the landing page</a> or use the menu at the top of the page to navigate to a specific area of the site. </p>
+
+      <div id="search-suggestions" style="display:none;">
+        <b>Maybe those suggestions help?</b>
+        <ul id="search-suggestions-list">
+          <li><a href="/">blablablabla</a></li>
```

Comment:
As discussed, moved to:

![image](https://cloud.githubusercontent.com/assets/1684420/19394091/8322e2dc-922f-11e6-8a27-df83485954d8.png)